test/test_sparse.py

```diff
@@ -2177,6 +2177,20 @@ def test_factory_empty_indices(self):
         expected_indices = torch.empty((4, 0), dtype=torch.long, device=device)
         self.assertEqual(tensor._indices(), expected_indices)
 
+    def test_sparse_abs(self):
+        device = 'cuda' if self.is_cuda else 'cpu'
+        i = torch.tensor([[0, 1, 1],
```

Comment:
Instead of creating one fixture, writing tests that can be run with EVERY unary op will allow us to invest more in each test and create multiple tensors to test with. For example, coalesced and uncoalesced sparse tensors.